docs/user-guide/walkthrough/README.md

```diff
@@ -26,54 +26,27 @@ For Kubernetes 101, we will cover kubectl, pods, volumes, and multiple container
 
 In order for the kubectl usage examples to work, make sure you have an examples directory locally, either from [a release](https://github.com/GoogleCloudPlatform/kubernetes/releases) or [the source](https://github.com/GoogleCloudPlatform/kubernetes).
 
-Table of Contents
-- [Kubectl CLI](#kubectl-cli)
-  - [Install Kubectl](#install-kubectl)
-  - [Configure Kubectl](#configure-kubectl)
-- [Pods](#pods)
-  - [Pod Definition](#pod-definition)
-  - [Pod Management](#pod-management)
-  - [Volumes](#volumes)
-  - [Multiple Containers](#multiple-containers)
-- [What's Next?](#whats-next)
-
+**Table of Contents**
+<!-- BEGIN MUNGE: GENERATED_TOC -->
+- [Kubernetes 101 - Kubectl CLI & Pods](#kubernetes-101---kubectl-cli-&-pods)
```

Comment:
If this link doesn't work out of the box, we should change the title so that it does.